tests/integration/targets/inventory_aws_ec2/playbooks/test_populating_inventory_with_include_or_exclude_filters.yml

```diff
@@ -0,0 +1,118 @@
+---
+- hosts: 127.0.0.1
+  connection: local
+  gather_facts: no
+  environment: "{{ ansible_test.environment }}"
+  tasks:
+
+    - module_defaults:
+        group/aws:
+          aws_access_key: '{{ aws_access_key }}'
+          aws_secret_key: '{{ aws_secret_key }}'
+          security_token: '{{ security_token | default(omit) }}'
+          region: '{{ aws_region }}'
+      block:
+
+        # Create VPC, subnet, security group, and find image_id to create instance
+
+        - include_tasks: setup.yml
+
+        # Create new host, refresh inventory
+        - name: create a new host (1/3)
+          ec2:
```

Comment:
The ec2 module is boto v2 based and handles rate-limiting poorly (resulting in test flakes).

I believe ec2_instance is on the priority list to migrate to amazon.aws I'd recommend using it instead.

Reply:
I just follow the de-facto convention for this test role. I would prefer to change that in another PR and take care of all the `ec2_instance` calls.

Reply:
FYI I've already made that change in community.aws